presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -154,41 +155,45 @@ public void testResetSession()
     @Test
     public void testCreateTable()
     {
-        assertUpdate("CREATE TABLE test_create (a bigint, b double, c varchar)");
-        assertTrue(getQueryRunner().tableExists(getSession(), "test_create"));
-        assertTableColumnNames("test_create", "a", "b", "c");
+        String tableName = getTestTableName("test_create");
```

Comment:
It looks like only few tests left... so maybe let's continue the convention from other tests.

Reply:
Ah, I thought I had gotten them all. I just added a few that I missed the first round, but let me know if I missed any more.